src/main/java/io/confluent/connect/jdbc/source/DataConverter.java

```diff
@@ -175,6 +181,23 @@ private static void addFieldSchema(ResultSetMetaData metadata, int col,
       }
 
       case Types.NUMERIC:
+        if (numericPrecisionMapping) {
+          int precision = metadata.getPrecision(col);
+          if (metadata.getScale(col) == 0 && precision < 20) { // integer
```

Comment:
Sorry, I do not follow this comment.
int8 (aka byte) = -128..127 that is 3 digits. How can we put it into NUMBER(2,0)?
similarly int16 (aka int) = -32768.. 32767 - 5 digits
...
int64 (aka long) = -9223372036854775808..9223372036854775807 - 19 digits